akka-docs/src/main/paradox/remoting-artery.md

```diff
@@ -102,7 +106,27 @@ listening for connections and handling messages as not to interfere with other a
 @@@
 
 The example above only illustrates the bare minimum of properties you have to add to enable remoting.
-All settings are described in [Remote Configuration](#remote-configuration-artery).
+All settings are described in @ref:[Remote Configuration](#remote-configuration-artery).
+
+### Selecting transport
+
+There are three alternatives of which underlying transport to use. It is configured by property
+`akka.remote.artery.transport` with the possible values:
+
+* `aeron-udp` - Based on [Aeron (UDP)](https://github.com/real-logic/aeron)
+* `tcp` - Based on @ref:[Akka Streams TCP](stream/stream-io.md#streaming-tcp)
+* `tls-tcp` - Same as `tcp` with encryption using @ref:[Akka Streams TLS](stream/stream-io.md#tls)
+
+The Aeron (UDP) transport is a high performance transport and should be used for systems
+that require high throughput and low latency. It is using more CPU resources than TCP
+when the system is idle or at low message rates. There is no encryption for Aeron.
+
+The TCP and TLS transport is implemented using Akka Streams TCP/TLS. This is the choice
+when encryption is needed, but it can also be used with plain TCP without TLS.
+It has very good performance (high throughput and low latency) but not as good as the Aeron transport.
+It is using less CPU resources than Aeron when the system is idle or at low message rates.
+This has not been verified yet, but it might scale better for many connections than the Aereon
```

Comment:
What is the reasoning behind this? Isn't the TCP 3x per pair of nodes whereas Aeron will be UDP as needed?

Reply:
yeah, perhaps not mention this aspect at all, since we haven't verified with many connections.

The Aeron "limitation" was only based on what is mentioned in the docs here: https://github.com/real-logic/aeron/wiki/Best-Practices-Guide#system-design
We also know that it's requiring much memory, increasing with number of channels/streams.